### PR TITLE
make llm imports optional

### PR DESCRIPTION
**Pull request by hwchase17**

### CODE REVIEW

File: libs/langchain/tests/unit_tests/llms/test_imports.py [hunk 1]
```diff
@@ -0,0 +1,9 @@
+from langchain import llms
+from langchain.llms.base import BaseLLM
+
+
+def test_all_imports() -> None:
+    """Simple test to make sure all things can be imported."""
+    for cls in llms.__all__:
+        exec(f"from langchain.llms import {cls}")
```
Comment (eyurtsev) [inline libs/langchain/tests/unit_tests/llms/test_imports.py:7]: Let's use importlib here?

File: libs/langchain/tests/unit_tests/llms/test_imports.py [hunk 2]
```diff
@@ -0,0 +1,8 @@
+from langchain import llms
+
+
+def test_all_imports() -> None:
+    """Simple test to make sure all things can be imported."""
+    for cls in llms.__all__:
+        exec(f"from langchain.llms import {cls}")
+        # TODO: assert all is of type BaseLLM
```
Comment (nfcampos) [inline libs/langchain/tests/unit_tests/llms/test_imports.py:8]: lol

We could also use importlib for this, but given this is a test don't care either way

**Review by nfcampos — APPROVED** [review]

File: libs/langchain/tests/unit_tests/llms/test_imports.py (hunk 1 above)
Comment (eyurtsev) [reply libs/langchain/tests/unit_tests/llms/test_imports.py:7]: Actually I think getattr is enough

```
for cls in llms.__all__:
  assert isinstance(getattr(llms, cls), ...)
```

File: libs/langchain/tests/unit_tests/llms/test_imports.py (hunk 2 above)
Comment (eyurtsev) [reply libs/langchain/tests/unit_tests/llms/test_imports.py:8]: I don't think we should use exec -- automated tools aren't gonna exercise discretion about where `exec` is located

**Review by hinthornw — APPROVED** [review]

**Pull request merged**